src/hooks/usePermit2.ts

```diff
@@ -81,9 +81,8 @@ export default function usePermit(amount?: CurrencyAmount<Token>, spender?: stri
 
   // Permit2 should be marked syncing from the time approval is submitted (pending) until it is
   // synced in tokenAllowance, to avoid re-prompting the user for an already-submitted approval.
-  // It should *not* be marked syncing if not permitted, because the user must still take action.
   const [syncState, setSyncState] = useState(SyncState.SYNCED)
-  const isSyncing = isPermitted || isSigned ? false : syncState !== SyncState.SYNCED
+  const isSyncing = syncState !== SyncState.SYNCED
```

Comment:
`isSynced`

Reply:
`!==` - this is for `isSyncing`, because all of the downstream checks check for `isApprovalPending`